docker-id/index.md

```diff
@@ -0,0 +1,50 @@
+---
+description: Sign up for a Docker ID and log in
+keywords: accounts, docker ID, billing, paid plans, support, Cloud, Hub, Store, Forums, knowledge base, beta access
+title: Docker ID accounts
+---
+
+Your free Docker ID grants you access to Docker services such as the Docker
+Store, Docker Cloud, Docker Hub repositories, and some beta programs. Your
+Docker ID becomes repository namespace used by hosted services such as Docker
+Hub and Docker Cloud. All you need is an email address.
+
+This account also allows you to log in to services such as the Docker Support
+Center, the Docker Forums, and the Docker Success portal.
+
+
+## Register for a Docker ID
+
+Your Docker ID becomes your user namespace for hosted Docker services, and becomes your username on the Docker Forums. A Docker ID also
+
+1. Go to the [Docker Cloud sign up page](https://cloud.docker.com).
+
+2. Enter a username, a valid email address, and a password.
+
+3. Click **Sign Up**.
+
+   Docker sends a verification email to the address you provided.
+
+4. Click the link in the email to verify your address.
+
+> **Note**: You cannot log in with your Docker ID until you verify your email address.
+
+
+## Log in
+
+Once you register and verify your Docker ID email address, you can log in
+to Docker services.
+
+For Docker Cloud, Hub, and Store, log in using the web interface.
+
+![Login using the web interface](images/login-cloud.png)
+
+You can also log in using the the `docker login` command. (You can read more about `docker login` [here](../engine/reference/commandline/login/).)
+
+> **Note:** When you use the `docker login` command, your credentials are stored
+in your home directory in `.docker/config.json`. The password is hashed in this
+file.
+
+## The Accounts API
+
+Once you create and verify your Docker ID, you can also update it using the [Docker Accounts API](../engine/reference/api/docker_io_accounts_api/).
```

Comment:
Change to `/docker-hub/accounts-api/` per PR #606 from @bfirsh -- it's likely that we're merging that before this.